docs/cli/plugin_implementation_guide.md

```diff
@@ -26,6 +26,8 @@ Current implementations:
 
 ### Repository
```

Comment:
Maybe the section should be somehow described as "Legacy" or similar? (Either by updating the heading, or by adding separate Legacy section and making this a subheading under that?)